base/Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN apk add --update --no-cache \
     bash \
     bash-completion \
     bash-doc \
+    mariadb-connector-c \
```

Comment:
Can I be annoying and ask you place this in order with the other mariadb package please @AndreasMurk